From c55cb04384a190aaae5784f4ddc353e05f5a40eb Mon Sep 17 00:00:00 2001
From: Elliott Sales de Andrade <quantum.analyst@gmail.com>
Date: Tue, 5 Jul 2022 23:10:13 -0400
Subject: [PATCH] Split styling dependencies out of test requirements

Signed-off-by: Elliott Sales de Andrade <quantum.analyst@gmail.com>
---
 setup.cfg                  | 7 +++++--
 1 files changed, 7 insertions(+), 4 deletions(-)

diff --git a/setup.cfg b/setup.cfg
index 9354f9e..19f5d70 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -43,10 +43,13 @@ bokeh =
 	selenium
 test = 
-	flake8
-	isort
 	matplotlib
 	Pillow
 	pytest
+dev = 
+	flake8
+	isort
+	%(docs)s
+	%(test)s
 
 [options.packages.find]
 where = lib
-- 
2.36.1

